docs/user/readme.adoc

```diff
@@ -108,18 +109,29 @@ Here are some useful self-diagnostic commands:
 * To log all LSP requests, add `"rust-analyzer.trace.server": "verbose"` to the settings and look for `Server Trace` in the panel.
 * To enable client-side logging, add `"rust-analyzer.trace.extension": true` to the settings and open the `Console` tab of VS Code developer tools.
 
-=== Language Server Binary
+=== rust-analyzer Language Server Binary
 
 Other editors generally require the `rust-analyzer` binary to be in `$PATH`.
 You can download the pre-built binary from the https://github.com/rust-analyzer/rust-analyzer/releases[releases] page. Typically, you then need to rename the binary for your platform, e.g. `rust-analyzer-mac` if you're on Mac OS, to `rust-analzyer` and make it executable in addition to moving it into a directory in your `$PATH`.
 
+On Linux to install the `rust-analyzer` binary into `~/.cargo/bin` which usually is already added to `$PATH`, this commands could be used
```

Comment:
I don't think putting binaries not installed by cargo into `~/.cargo/bin` is a good idea.

Reply:
What fo you think is a better place?

Reply:
Maybe `~/.local/bin`? There are a lot of programs that install themselves there. It isn't in `PATH` by default though.

Reply:
Thanks. I also think this is a better suggestion. I changed it.